### ISSUE TITLE
Fine-tuning BERT using Next sentence prediction loss

### ISSUE THREAD

**Issue by JiajunBao** [huggingface/transformers#1622]
In `pytorch_pretrained_bert`, there is an example for fine-tuning BERT using next sentence prediction loss. In the new version, how shall we fine-tune BERT on the next sentence prediction task?

Thank you.

**Comment by LysandreJik** [comment]
We do not have any scripts that display how to do next sentence prediction as it was shown with RoBERTa to be of little importance during training.

We had some scripts until version 1.1.0 that allowed this, you can find them [here](https://github.com/huggingface/transformers/tree/1.1.0/examples/lm_finetuning). They are deprecated but can give you an idea of the process.

**Comment by JiajunBao** [comment]
Ah, gotcha. Thanks!
